Dockerfile.2.4.1

```diff
@@ -0,0 +1,187 @@
+FROM lambci/lambda:build-python3.7 
```

Comment:
As stated in my comments in `.drone.yml` I would suggest to only have one `Dockerfile` and build versions out of that when a git tag is created. That is in line how _releases_ are typically made I think.